docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:14.18.0-stretch-slim
+
+ARG CRAFTER_VERSION=latest
+ARG BUILD_DATE
+
+LABEL org.opencontainers.image.title="Crafter" \
+      org.opencontainers.image.description="An image to launch \"@funboxteam/crafter\" npm package" \
+      org.opencontainers.image.vendor="FunBox" \
+      org.opencontainers.image.licenses="MIT" \
```

Comment:
At first, we had been using MIT license for our open-source projects, but later we switched to Apache-2.0 (see https://github.com/funbox/blueprinter).

Should we switch to Apache-2.0 here too?

Reply:
We'd prefer to use Apache-2.0 for our opensource projects. If it's possible to switch here, let's do it.

Reply:
However, let's switch the license using a separate PR. 

It feels like a major change.

Reply:
https://github.com/funbox/crafter/pull/14